src/material/autocomplete/autocomplete-trigger.ts

```diff
@@ -879,13 +880,29 @@ export class MatAutocompleteTrigger
     });
   }
 
+  /** Implements BreakpointObserver to be used to detect handset landscape */
+  private _breakpointObserver = inject(BreakpointObserver);
+
   private _getOverlayPosition(): PositionStrategy {
+    // Set default Overlay Position
     const strategy = this._overlay
       .position()
       .flexibleConnectedTo(this._getConnectedElement())
       .withFlexibleDimensions(false)
       .withPush(false);
 
+    // Check breakpoint if being viewed in HandsetLandscape
+    const isHandsetLandscape = this._breakpointObserver
```

Comment:
Isn't this a subscription object not a boolean? Also I think we need to unsubscribe / takeUntil(destroyed)

Reply:
@mmalerba Thanks for the note! I've made the change to unsubscribe in my latest commit. Let me know if it's good to go and I'll make the request to merge.